services/shhext/service_test.go

```diff
@@ -104,6 +104,15 @@ func (s *ShhExtSuite) SetupTest() {
 		s.nodes[i] = stack
 	}
 	s.services[0].tracker.handler = newHandlerMock(1)
+
+	// Cleanup db file from previous runs
+	dbFilePath := fmt.Sprintf("%s/1.v2.db", os.TempDir())
```

Comment:
this is not what i meant though. right now this test is not really isolated. for example two tests will conflict with each other. or leftover from previous test run may impact with new run. what i suggested is to use **ioutill**.TempDir, it will create a directory with random prefix instead of using root temporary directory

Reply:
oh I see, I misunderstood, I can change that, thanks